analytics-core/src/main/java/com/segment/analytics/messages/Message.java

```diff
@@ -20,9 +20,11 @@
  * <a href="http://bit.ly/1N5v3Cq">Javadocs</a>.
  */
 public interface Message {
+  int MAX_MESSAGE_ID_LENGTH = 256;
```

Comment:
Wondering why this limit? I don't think there's anything in Segment with this restriction. Warehouse columns are capped at 256 by default but can be increased upon request